package.nls.json

```diff
@@ -120,6 +120,7 @@
     "AWS.samcli.deploy.stackName.error.invalidCharacters": "A stack name may contain only alphanumeric characters (case sensitive) and hyphens",
     "AWS.samcli.deploy.stackName.error.firstCharacter": "A stack name must begin with an alphabetic character",
     "AWS.samcli.deploy.stackName.error.length": "A stack name must not be longer than 128 characters",
+    "AWS.samcli.deploy.statusbar.message": "$(cloud-upload) Deploying SAM Application to {0}...",
```

Comment:
What does the syntax `$(cloud-upload)` do?

Reply:
It renders the icon you can see in the PR description screenshot
info: https://code.visualstudio.com/api/references/icons-in-labels